tools/bioformats2raw/bf2raw.xml

```diff
@@ -49,7 +49,7 @@ $bf2raw_params.droptop
     <inputs>
         <section name="io_options" title="Input-output paths">
```

Comment:
```suggestion
        <section name="io_options" title="Input-output paths" expanded="true">
```

Reply:
I think at least the input data should be visible by default immediately after opening the tool, since this will need to be set every time the tool is used, right?